cmd/config-migrate_test.go

```diff
@@ -191,172 +62,123 @@ func TestServerConfigMigrateV2toV33(t *testing.T) {
 	}
-	if err := migrateV12ToV13(); err == nil {
+	if _, err := migrateV12ToV13(configBytes); err == nil {
 		t.Fatal("migrateConfigV12ToV13() should fail with a corrupted json")
 	}
-	if err := migrateV13ToV14(); err == nil {
+	if _, err := migrateV13ToV14(configBytes); err == nil {
 		t.Fatal("migrateConfigV13ToV14() should fail with a corrupted json")
 	}
-	if err := migrateV14ToV15(); err == nil {
+	if _, err := migrateV14ToV15(configBytes); err == nil {
 		t.Fatal("migrateConfigV14ToV15() should fail with a corrupted json")
 	}
-	if err := migrateV15ToV16(); err == nil {
+	if _, err := migrateV15ToV16(configBytes); err == nil {
 		t.Fatal("migrateConfigV15ToV16() should fail with a corrupted json")
 	}
-	if err := migrateV16ToV17(); err == nil {
+	if _, err := migrateV16ToV17(configBytes); err == nil {
 		t.Fatal("migrateConfigV16ToV17() should fail with a corrupted json")
 	}
-	if err := migrateV17ToV18(); err == nil {
+	if _, err := migrateV17ToV18(configBytes); err == nil {
 		t.Fatal("migrateConfigV17ToV18() should fail with a corrupted json")
 	}
-	if err := migrateV18ToV19(); err == nil {
+	if _, err := migrateV18ToV19(configBytes); err == nil {
 		t.Fatal("migrateConfigV18ToV19() should fail with a corrupted json")
 	}
-	if err := migrateV19ToV20(); err == nil {
+	if _, err := migrateV19ToV20(configBytes); err == nil {
 		t.Fatal("migrateConfigV19ToV20() should fail with a corrupted json")
 	}
-	if err := migrateV20ToV21(); err == nil {
+	if _, err := migrateV20ToV21(configBytes); err == nil {
 		t.Fatal("migrateConfigV20ToV21() should fail with a corrupted json")
 	}
-	if err := migrateV21ToV22(); err == nil {
+	if _, err := migrateV21ToV22(configBytes); err == nil {
 		t.Fatal("migrateConfigV21ToV22() should fail with a corrupted json")
 	}
-	if err := migrateV22ToV23(); err == nil {
+	if _, err := migrateV22ToV23(configBytes); err == nil {
 		t.Fatal("migrateConfigV22ToV23() should fail with a corrupted json")
 	}
-	if err := migrateV23ToV24(); err == nil {
+	if _, err := migrateV23ToV24(configBytes); err == nil {
 		t.Fatal("migrateConfigV23ToV24() should fail with a corrupted json")
 	}
-	if err := migrateV24ToV25(); err == nil {
+	if _, err := migrateV24ToV25(configBytes); err == nil {
 		t.Fatal("migrateConfigV24ToV25() should fail with a corrupted json")
 	}
-	if err := migrateV25ToV26(); err == nil {
+	if _, err := migrateV25ToV26(configBytes); err == nil {
 		t.Fatal("migrateConfigV25ToV26() should fail with a corrupted json")
 	}
-	if err := migrateV26ToV27(); err == nil {
+	if _, err := migrateV26ToV27(configBytes); err == nil {
 		t.Fatal("migrateConfigV26ToV27() should fail with a corrupted json")
 	}
-	if err := migrateV27ToV28(); err == nil {
+	if _, err := migrateV27ToV28(configBytes); err == nil {
+		t.Fatal("migrateConfigV27ToV28() should fail with a corrupted json")
+	}
+	if _, err := migrateV28ToV29(configBytes); err == nil {
+		t.Fatal("migrateConfigV27ToV28() should fail with a corrupted json")
```

Comment:
migrateConfigV27ToV28 -> migrateConfigV28ToV29. The error messages need to reflect the correct migration versions for this and subsequent errors